src/launch_common.py

```diff
@@ -83,18 +83,21 @@ def build_parser():
 
 def add_argument_cycle(parser):
     parser.add_argument("-C", "--initial_cycle",
-                        help="First cycle to start payment. For last released rewards, set to 0. Non-positive values "
-                             "are interpreted as: current cycle - abs(initial_cycle) - (NB_FREEZE_CYCLE+1) - release_override. "
-                             "If not set application will continue from last payment made or last reward released.",
-                        type=int)
+                        help="First cycle to start payment(s) from depending on run mode -M. "
+                             "The default value is 0, which will start at the most recently released reward. "
+                             "Positive values sets desired cycle directly: initial_cycle <= current cycle - (NB_FREEZE_CYCLE+1) - release_override. "
+                             "Non-positive values count backwards: current cycle - abs(initial_cycle) - (NB_FREEZE_CYCLE+1) - release_override.",
+                        default=0, type=int)
```

Comment:
@nicolasochem I added the default value here as all other options have them here as well. I tried to adapt the help section of -C and the confusing part is that it does not work consistently in the ranges from negativ over 0 to positive values. What do you think?

Reply:
In addition I think we have to restrict the value range of release_override from [-11,-1] as it crosses with the initial_cycle defintion, or?

Reply:
@jdsika I see what you tried to do by fixing the help message but I think it fell short. I pushed another commit on top of yours. I think it makes the help message more helpful, please let me know what you think. Please merge if it's ok.

>In addition I think we have to restrict the value range of release_override from [-11,-1] as it crosses with the initial_cycle defintion, or?

Yes, it's probably out of scope for this PR but should be done.

Reply:
I just added it. Otherwise we will forget about it. I noted it in the help section and extended the value check after parsing.